### PR TITLE
docs(fix): various wordings - Mta-1582

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: dedibox-network/domains/how-to/renew-domain.mdx [hunk 1]
```diff
@@ -41,6 +41,6 @@ When you purchase a domain name, you can use it for the period of time you regis
   You are redirected to the payment gateway to complete your payment.
 
 <Navigation title="See Also">
-  <PreviousButton to="/dedibox-network/domains/how-to/purchase-domain/">How to purchase a domain name</PreviousButton>
+  <PreviousButton to="/dedibox-network/domains/how-to/buy-domain/">How to buy a domain name</PreviousButton>
```
Comment (RoRoJ) [inline dedibox-network/domains/how-to/renew-domain.mdx:44]: do we definitely want to change slugs too? do we need redirects after?

**Review by RoRoJ — APPROVED** [review]

File: dedibox-network/domains/how-to/transfer-domain.mdx [hunk 2]
```diff
@@ -30,7 +30,7 @@ You can transfer any domain purchased or renewed at another registrar, to Scalew
 
 1. In your browser, navigate to [the Scaleway domain name order page](https://www.scaleway.com/en/domain/).
 
-2. Enter your domain name in the search bar, and click **search a domain**. The **Domain name choice** page displays, showing that the domain is not available for purchase.
+2. Enter your domain name in the search bar, and click **search a domain**. The **Domain name choice** page displays, showing that the domain is not available for buy.
```
Comment (ldecarvalho-doc) [inline dedibox-network/domains/how-to/transfer-domain.mdx:33]: the button wording is in all caps, no?

File: dedibox-network/domains/how-to/transfer-domain.mdx [hunk 3]
```diff
@@ -52,7 +52,7 @@ You can transfer any domain purchased or renewed at another registrar, to Scalew
 
   <Message type="note">
 
-  If you purchase web hosting, you get your domain name for free with purchase / renewal.
+  If you buy web hosting, you get your domain name for free with your order / renewal.
```
Comment (ldecarvalho-doc) [inline dedibox-network/domains/how-to/transfer-domain.mdx:55]:
```suggestion
  If you buy web hosting, you get your domain name for free with your order or renewal.
```

File: dedibox-network/domains/quickstart.mdx [hunk 4]
```diff
@@ -48,7 +48,7 @@ A domain name is the equivalent of your postal address on the web, and allows yo
 
   <Message type="note">
 
-  If you purchase web hosting, you get your domain name for free with purchase/renewal.
+  If you buy web hosting, you get your domain name for free with your order/renewal.
```
Comment (ldecarvalho-doc) [inline dedibox-network/domains/quickstart.mdx:51]:
```suggestion
  If you buy web hosting, you get your domain name for free with your order or renewal.
```

File: dedibox-network/domains/how-to/transfer-domain.mdx (hunk 2 above)
Comment (bene2k1) [reply dedibox-network/domains/how-to/transfer-domain.mdx:33]: no its all lowercases :(

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**